### DISCUSSION TITLE
Question about grpc in mojo

### DISCUSSION THREAD

**Discussion by liuzhishan** [modular/modular#3132] [Q&A]
Hi, I'm new to mojo, and I have a question: how can I use grpc in mojo? 

It seems to be a common situation in distributed training. Like in tensorflow, I can write an op kernel using c++, and then send some grpc request to other machine and get result, then write the result to tensor. The tensor will be used in tensorflow graph written by python code. How I achieve this functionality in mojo ?

**Comment by Brian-M-J** [comment]
Someone would have to make a Mojo wrapper for the [core C++ library](https://github.com/grpc/grpc). I don't know how far Mojo's C++ interop has come, so I don't know if this is possible yet.

**Reply by liuzhishan** [reply]
Thanks for reply. I didn't find document about the progress of c++ interop neither.
